control/sisotool.py

```diff
@@ -1,5 +1,10 @@
 __all__ = ['sisotool', 'rootlocus_pid_designer']
 
+import numpy as np
+import matplotlib.pyplot as plt
+import warnings
+from functools import partial
```

Comment:
Worth running this throug `isort`?

`warnings` and `functools` are stdlib, Numpy and Pyplot external.